src/main.rs

```diff
@@ -20,7 +20,7 @@ async fn main() -> ExitCode {
 #[allow(clippy::print_stderr)]
 async fn run_sync() -> Result<()> {
 	let config = {
-		let config_path = std::env::var("FAMEDLY_LDAP_SYNC_CONFIG").unwrap_or("config.yaml".into());
+		let config_path = std::env::var("FAMEDLY_SYNC_CONFIG").unwrap_or("config.yaml".into());
```

Comment:
This is a breaking change and the commit at least needs to be marked as such. Not 100% sure it's worth doing.

Reply:
(Also, we can address this in a branch/issue separate from the v2 api/cache-removal branch, so maybe we shouldn't do this here)

Reply:
Wait, I believe this is just a revert of some mistaken rebase. We have FAMEDLY_SYNC_CONFIG on main:
https://github.com/famedly/famedly-sync/blob/main/src/main.rs#L23